src/main/webapp/app/types/sidebar.ts

```diff
@@ -91,6 +94,34 @@ export interface SidebarCardElement {
      * Set for Exercises. Will be removed after refactoring
      */
     exercise?: Exercise;
+    // TODO Extra Exam Part
```

Comment:
If the 2 TODOs shall be left in, could you please specify what shall be done? That is not clear for me

Reply:
Forgot to remove it, thanks 👍